prelude/bootstrap.js

```diff
@@ -1291,10 +1319,6 @@ function payloadFileSync(pointer) {
     // For newer node versions (after https://github.com/nodejs/node/pull/33229 ):
     // Returns an array [string, boolean].
     //
-    var returnArray =
```

Comment:
?? Why this is removed?

Reply:
Because linter pointed out that the returnArray is not used anywhere later on

Reply:
miss merge..